crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -220,6 +224,76 @@ impl GlobalTransform {
         }
     }
 
+    /// Divide `self` with `transform`, this is the reciprocal of [`Self::mul_transform`].
+    ///
+    /// `t2 * t1 / t2 == t1` Note that transforms are not commutative, meaning that
+    /// `t1 * t2 / t2 != t1`.
+    /// ```rust
+    /// # use bevy_math::{Vec3, Quat};
+    /// # use bevy_transform::prelude::{GlobalTransform, Transform};
+    /// fn identity(t1: GlobalTransform, t2: GlobalTransform) -> Transform {
+    ///     // Equivalent to `t2 * t1 / t2`
+    ///     t2.mul_transform(t1.into()).div_transform(t2)
+    /// }
+    /// let t1 = GlobalTransform {
+    ///     translation: Vec3::new(1034.0, 34.0, -1324.34),
+    ///     rotation: Quat::from_rotation_x(1.2),
+    ///     scale: Vec3::new(1.0, 2.345, 0.9),
+    /// };
+    /// let t2 = GlobalTransform {
+    ///     translation: Vec3::new(28.0, -54.493, 324.34),
+    ///     rotation: Quat::from_rotation_z(1.9),
+    ///     scale: Vec3::new(3.0, 1.345, 0.9),
+    /// };
+    /// let f32_equal = |left: f32, right: f32| (left - right).abs() < 0.0001;
+    /// let rt_t1_pos = identity(t1, t2).translation;
+    /// let rt_t2_pos = identity(t2, t1).translation;
+    /// assert!(f32_equal(t1.translation.length(), rt_t1_pos.length()));
+    /// assert!(f32_equal(t2.translation.length(), rt_t2_pos.length()));
+    /// assert!(f32_equal(t1.scale.length(), identity(t1, t2).scale.length()));
+    /// assert!(f32_equal(t2.scale.length(), identity(t2, t1).scale.length()));
+    /// ```
```

Comment:
this feels like it would be better as an actual test than a doc test

Reply:
Good suggestion. Should be good now.